Formula/g/go@1.24.rb

```diff
@@ -88,7 +88,7 @@ def install
 
     # Try running a sample using cgo without CC or CXX set to ensure that the
     # toolchain's default choice of compilers work
-    with_env(CC: nil, CXX: nil) do
+    with_env(CC: nil, CXX: nil, CGO_ENABLED: "1") do
```

Comment:
there are some discussions about this in https://github.com/Homebrew/homebrew-core/pull/249408